libvcx/src/connection.rs

```diff
@@ -1353,37 +1358,38 @@ pub mod tests {
 
     pub fn create_connected_connections() -> (u32, u32) {
```

Comment:
This is used only be tests `agency_pool_tests` tests. Those tests are still not passing due to some issues, however this is their shared step and changes in this PR fixes run of `create_connected_connections`. Previously this method assumed legacy connections